programming/sed-article.md

```diff
@@ -324,24 +307,23 @@ escape them with a backslash (`\(` and `\)`).
 By default regular expressions match any text that contains a match.
 For example, `/two/` matches "two", "one two", "two three", and "one two three".
 
-To require the match to appear at the beginning of the text,
-precede the pattern with `^`. For example, `/^two/`.
+To require the match to appear at the beginning of a line,
```

Comment:
What is a "line" in the context of regular expression input?
It seems better to say "text".

Reply:
There is ambiguity both ways.  When I think of text, I think of the whole stream of text that is being processed.  Line is ambiguous because the pattern space may contain multiple lines.  Take your pick.